### PR TITLE
Updated ecommerce API requirement from 0.4.0 to 1.0.0

### PR DESCRIPTION
**Pull request by Nickersoft**
@clintonb @rlucioni FYI: @jimabramson @wedaly

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -17,4 +17,4 @@ paypalrestsdk==1.9.0
 pycountry==1.10
 requests==2.6.0
 
-git+https://github.com/edx/ecommerce-api-client.git@0.4.0#egg=ecommerce-api-client==0.4.0
+git+https://github.com/edx/ecommerce-api-client.git@1.0.0#egg=ecommerce-api-client==1.0.0
```
Comment (clintonb) [inline requirements/base.txt]: Please move this to `test.txt` since this is only needed for tests.

**Comment by clintonb** [comment]
:+1: (after moving)

File: requirements/base.txt (hunk 1 above)
Comment (Nickersoft) [inline requirements/base.txt]: @clintonb The entire line, or just the updated line?

Comment (rlucioni) [inline requirements/base.txt]: @Nickersoft this requirement (the ecommerce API client).

**Comment by rlucioni** [comment]
:+1: once the build passes.

**Comment by jimabramson** [comment]
what they said. :+1:

**Pull request merged**